code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -35,33 +34,32 @@ datum/reagent/styptic_powder
 	id = "styptic_powder"
 	description = "100% chance per cycle of healing 2 points of BRUTE damage."
 	reagent_state = LIQUID
-	color = "#C8A5DC" // rgb: 200, 165, 220
+	color = "#FF9696"
 	metabolization_rate = 2
 
 datum/reagent/styptic_powder/reaction_mob(var/mob/living/M as mob, var/method=TOUCH, var/volume)
 	if(method == TOUCH)
-		M.heal_organ_damage(volume*REM,0)
-		M << "<span class='notice'>You feel your wounds knitting back together!</span>"
+		M.adjustBruteLoss((volume-(volume*1))*REM)
+		M << "<span class='notice'>The styptic powder stings like hell as it closes some of your wounds.</span>"
 		M.emote("scream")
 	if(method == INGEST)
 		M.adjustToxLoss(0.5*volume)
-		M << "<span class='notice'>You probably shouldn't have eaten that. Maybe you should of splashed it on, or applied a patch?</span>"
+		M << "<span class='notice'>You feel gross!</span>"
 	..()
 	return
 
 datum/reagent/styptic_powder/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	if(prob(55))
-		M.adjustBruteLoss(-2*REM)
+		M.adjustBruteLoss(-8*REM)
 	..()
```

Comment:
This needs to retain the probability to heal 8; otherwise a single patch is going to heal 100 damage.